src/packages/fetch-engine/src/__tests__/download.test.ts

```diff
@@ -327,14 +327,16 @@ describe('download', () => {
         'rhel-openssl-1.1.x',
         'windows',
         'linux-musl',
+        'freebsd12',
```

Comment:
We do not provide binaries for freebsd12, why did you add it? 🤔

Reply:
To make sure our behavior there is consistent.
But as for this test it can indeed be removed